src/main/java/commands/UserCommand.java

```diff
@@ -0,0 +1,17 @@
+package commands;
+
+import tasks.TaskList;
+import exceptions.DukeException;
+
+public abstract class UserCommand {
+    TaskList tasks;
+
+    UserCommand(){}
```

Comment:
Not sure what the purpose of this is

Reply:
Ethan, thanks for your comment!
Because some of its subclasses require no-argument constructor, I want to avoid constructor chaining here.